src/test/java/com/sonyericsson/jenkins/plugins/bfa/DisplayDownstreamTest.java

```diff
@@ -204,11 +260,11 @@ private void createMatrixProjectNews() throws Exception {
                 PROJECT_NE, PROJECT_EW, PROJECT_SE);
         createFreestyleProjectWithShell(PROJECT_SW, "rapakalja");
 
-        hudson.setNumExecutors(50);
+        jenkins.getInstance().setNumExecutors(50);
```

Comment:
Ah, I've never used `getInstance()` myself before so I just assumed it did something else :confused: So never mind, move along :)